package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -476,6 +476,7 @@ dhcp_add() {
 	config_get leasetime "$cfg" leasetime 12h
 	config_get options "$cfg" options
 	config_get_bool dynamicdhcp "$cfg" dynamicdhcp 1
+	config_get tag "$cfg" tag
```

Comment:
Handle tag parameter as a list as dnsmasq allows more than one tag to be matched